docs/tendermint-core/validators.md

```diff
@@ -17,14 +17,73 @@ to post any collateral at all.
 
 Validators have a cryptographic key-pair and an associated amount of
 "voting power". Voting power need not be the same.
+TestDuplicateVoteEvidenceValidation
 
 ## Becoming a Validator
 
 There are two ways to become validator.
 
-1.  They can be pre-established in the [genesis state](./using-tendermint.md#genesis)
-2.  The ABCI app responds to the EndBlock message with changes to the
-    existing validator set.
+1. They can be pre-established in the [genesis state](./using-tendermint.md#genesis)
+2. The ABCI app responds to the EndBlock message with changes to the
+   existing validator set.
+
+## Setting up a Validator
+
+When setting up a validator there are countless ways to configure your setup. This guide is aimed at showing one of them, the sentry node design. This design is mainly for DDOS prevention.
+
+#### Network Layout
+
+![ALT Network Layout](./sentry_layout.png)
+
+The diagram is based on AWS, other cloud providers will have similar solutions to design a solution.
+
+The proposed network diagram is similar to the classical backend/frontend separation of services in a corporate environment. The “backend” in this case is the private network of the validator in the data center. The data center network might involve multiple subnets, firewalls and redundancy devices, which is not detailed on this diagram. The important point is that the data center allows direct connectivity to the chosen cloud environment. Amazon AWS has “Direct Connect”, while Google Cloud has “Partner Interconnect”. This is a dedicated connection to the cloud provider (usually directly to your virtual private cloud instance in one of the regions).
+
+All sentry nodes (the “frontend”) connect to the validator using this private connection. The validator does not have a public IP address to provide its services.
+
+Amazon has multiple availability zones within a region. One can install sentry nodes in other regions too. In this case the second, third and further regions need to have a private connection to the validator node. This can be achieved by VPC Peering (“VPC Network Peering” in Google Cloud). In this case, the second, third and further region sentry nodes will be directed to the first region and through the direct connect to the data center, arriving to the validator.
+
+A more persistent solution (not detailed on the diagram) is to have multiple direct connections to different regions from the data center. This way VPC Peering is not mandatory, although still beneficial for the sentry nodes. This overcomes the risk of depending on one region. It is more costly.
```

Comment:
Should we also say non-cloud setup is also possible (i.e. bare-metal)? ^ implies that cloud is the only option

Reply:
added to the first paragraph in this section.